components/ButtonBar/BUILD

```diff
@@ -122,6 +122,7 @@ mdc_unit_test_suite(
         ":unit_test_sources",
         ":unit_test_swift_sources",
     ],
+    use_autobot_environment_runner = False,
```

Comment:
Please add a comment to this line to indicate that it's a temporary work-around.

Reply:
Addressed in 3bea593326398178327f82a9541b971d4cac9f1d.